packages/manager/apps/dedicated/client/app/cdn/dedicated/cdn-dedicated.service.js

```diff
@@ -137,7 +137,8 @@ angular
           }
           return $q.reject(cdn);
         })
-        .then(() => result, (http) => $q.reject(http.data));
+        .then(() => result)
+        .catch((http) => $q.reject(http.data));
```

Comment:
This doesn't seem related to remove `responsive-tabs`, if it's this type of change is only made once, it's ok but if it's done a few times myabe it should be brought by its own commit

Reply:
Done !